### ISSUE TITLE
Internal Server error, node.js Framework version 3.1.1

### ISSUE THREAD

**Issue by plkumar** [Microsoft/BotBuilder#811]
Receiving the following error when responding to a Skype user, same code works with  WebChat but messages are displayed only after a next message is sent from user side.

Error: Request to 'https://skype.botframework.com/v3/conversations/29%3A1lm8ymLQhQ5WvPYm55K2BAd5u6klEeaEJuZBvHXgskBI/activities' failed: [500] Internal Server Error
    at Request._callback (/Users/lakshman/Documents/StockBot/node_modules/botbuilder/lib/bots/ChatConnector.js:356:46)
    at Request.self.callback (/Users/lakshman/Documents/StockBot/node_modules/request/request.js:187:22)
    at emitTwo (events.js:100:13)
    at Request.emit (events.js:185:7)
    at Request.<anonymous> (/Users/lakshman/Documents/StockBot/node_modules/request/request.js:1044:10)
    at emitOne (events.js:90:13)
    at Request.emit (events.js:182:7)
    at IncomingMessage.<anonymous> (/Users/lakshman/Documents/StockBot/node_modules/request/request.js:965:12)
    at emitNone (events.js:85:20)
    at IncomingMessage.emit (events.js:179:7)

**Comment by msft-shahins** [comment]
What is the Microsoft App Id of your bot?

**Comment by plkumar** [comment]
here it is 30a5ed7b-8880-4025-b765-50094b992afc

**Comment by danmarshall** [comment]
@plkumar are you still seeing this error?

**Comment by plkumar** [comment]
@danmarshall 

yes, i'm still being this issue.

Error: Request to 'https://skype.botframework.com/v3/conversations/29%3A1lm8ymLQhQ5WvPYm55K2BAd5u6klEeaEJuZBvHXgskBI/activities' failed: [500] Internal Server Error
    at Request._callback (/Users/lakshman/Documents/StockBot/node_modules/botbuilder/lib/bots/ChatConnector.js:356:46)
    at Request.self.callback (/Users/lakshman/Documents/StockBot/node_modules/request/request.js:187:22)
    at emitTwo (events.js:100:13)
    at Request.emit (events.js:185:7)
    at Request.<anonymous> (/Users/lakshman/Documents/StockBot/node_modules/request/request.js:1044:10)
    at emitOne (events.js:90:13)
    at Request.emit (events.js:182:7)
    at IncomingMessage.<anonymous> (/Users/lakshman/Documents/StockBot/node_modules/request/request.js:965:12)
    at emitNone (events.js:85:20)
    at IncomingMessage.emit (events.js:179:7)

**Comment by danmarshall** [comment]
Can you please provide a small code sample which reproduces this?

**Comment by Andrea-Orimoto** [comment]
Closing due to inactivity, please open a new issue or reply to this one asking us to reopen if needed. Thanks!
